src/geodedic.rs

```diff
@@ -203,12 +226,35 @@ impl Coordinate<Ecef> {
 
     /// Converts an Earth-Centered, Earth-Fixed coordinate into latitude, longitude, and altitude.
     ///
-    /// Note that this conversion is not trivial and needs to be approximated. This implementation
-    /// currentaly uses [Ferrari's solution][ferrari], but this may change in the future.
+    /// Note that this conversion is not trivial and needs to be approximated.
+    ///
+    /// The implementation currently only guarantees conversion to Wgs84 datums with altitude
+    /// between -10km and 50km from the surface of the Wgs84 ellipsoid, roughly corresponding
+    /// to the bottom of the Mariana Trench to the top of the stratosphere. Outside this range,
+    /// the implementation may panic.
+    ///
+    /// This implementation currently uses [Ferrari's solution][ferrari], but this may change
+    /// in the future.
     ///
     /// [ferrari]: https://en.wikipedia.org/wiki/Geographic_coordinate_conversion#The_application_of_Ferrari's_solution
     #[must_use]
     pub fn to_wgs84(&self) -> Wgs84 {
+        let geo_center_distance_sq =
```

Comment:
Let's guard this whole check behind `if cfg!(debug_assertions)` so that it doesn't incur runtime cost in release mode.

Reply:
I did #[cfg(any(debug_assertions, test))]